src/library_dylink.js

```diff
@@ -171,8 +173,12 @@ var LibraryDylink = {
         // https://github.com/WebAssembly/mutable-global/issues/1
         value = value.value;
       }
+#if MEMORY64
+      if (typeof value == 'bigint') {
+#else
       if (typeof value == 'number') {
```

Comment:
Maybe we could add a helper,
```js
if (typeof value == {{{ JS_PTR_TYPE_NAME }}}) {
```
which is equal to either `'bigint', 'number'`? Would help in at least two places to avoid duplicated ifs.

Reply:
Good idea.. done.